packages/cms-migration/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
so you made this a separate package because in the end its all just throwaway code, yeah? I thought maybe it should be part of squidex package then. Just thinking loud..

Reply:
I thought about adding it to `package/squidex` or even `package/contentful`. But for me it seemed a little bit weird to install contentful libs in `package/squidex` (for example, contentful-management) or install `@asap/squidex` in `package/contentful`

Reply:
so I created a separate package

Reply:
yeah good point, this is fine then